### ISSUE TITLE
Rendering broken on latest 2.26.3

### ISSUE THREAD

**Issue by jyboudreau** [flathub/community.pathofbuilding.PathOfBuilding#39]
After the latest update the rendering is broken:

![Screenshot from 2023-04-08 11-19-34](https://user-images.githubusercontent.com/62624/230729268-4f1f7d1e-9900-45cd-9efb-d860aaa5816c.png)

Distro: Nobara 37
Gnome: 43.2
Windowing System: X11

**Comment by ernstp** [comment]
Does this work: https://github.com/flathub/community.pathofbuilding.PathOfBuilding/pull/1#issuecomment-1501093106
?

**Comment by jyboudreau** [comment]
Issue is fixed in latest version. I also tested the QT_SCALE_FACTOR which does scale things. However a few observations:

Any scaling creates a slightly blurred image. Even the default experience does this. I figured out that this is because I have `Large Text` turned on in accessibility settings (getting old 😄) which automatically picked up by QT

To have the "default" POB experience with text that's not blurry I need to set the following:

```
flatpak override --user  --env=QT_ENABLE_HIGHDPI_SCALING=0 community.pathofbuilding.PathOfBuilding
```

Which disables the auto scaling and reverts back to Qt 5 default behaviour. This makes the text its normal small size but more readable for me.

Thanks for the fix and for maintaining this project!
